arviz/plots/styles/arviz-grayscale.mplstyle

```diff
@@ -0,0 +1,38 @@
+# This style is based on Seaborn-white parameters, the main differences are
+# Default matplotlib font (no problem using Greek letters!)
+# Larger font size for several elements
+# Colorblind friendly color cycle
+figure.figsize: 7.2, 4.8
+figure.dpi: 100.0
+figure.facecolor: white
+text.color: .15
+axes.labelcolor: .15
+legend.frameon: False
+legend.numpoints: 1
+legend.scatterpoints: 1
+xtick.direction: out
+ytick.direction: out
+xtick.color: .15
+ytick.color: .15
+axes.axisbelow: True
+lines.solid_capstyle: round
+
+axes.labelsize: 15
+axes.titlesize: 16
+xtick.labelsize: 14
+ytick.labelsize: 14
+legend.fontsize: 14
+
+axes.grid: False
+axes.facecolor: white
+axes.edgecolor: 0
+axes.linewidth: 1
+axes.spines.top: False
+axes.spines.right: False
+image.cmap: gray
```

Comment:
what cmap is this? The mpl default or the one in colorcet (etc+)

Reply:
The default from matplotlib, as I understand this one is perceptually uniform. But I could check again.

Reply:
It is not.

Reply:
:( OK, I will fix it